sql/migrate/migrate.go

```diff
@@ -389,27 +389,18 @@ var _ Dir = (*LocalDir)(nil)
 var (
 	// templateFuncs contains the template.FuncMap for the DefaultFormatter.
 	templateFuncs = template.FuncMap{
-		"now": time.Now,
-		"sem": ensureSemicolonSuffix,
+		"now": func() string { return time.Now().Format("20060102150405") },
```

Comment:
Maybe just introduce a const for this format? so it is more clear and add a small comment

Reply:
Since it is only used in one place I don't think adding a const is necessary. I added a comment explaining this choice though.